platforms/documentation/docs/src/docs/userguide/optimizing-performance/build-cache/build_cache.adoc

```diff
@@ -15,6 +15,15 @@
 [[build_cache]]
 = Build Cache
 
+++++
+<div class="badge-wrapper">
+    <a class="badge" href="https://dpeuniversity.gradle.com/app/courses/54469478-55ba-416d-9cef-3b5aa33dd2a5/" target="_blank">
+        <span class="badge-type button--blue">LEARN</span>
+        <span class="badge-text">Gradle Build Cache Deep Dive&nbsp;&nbsp;&nbsp;&gt;</span>
```

Comment:
Given the content of this doc page - should this point to the incremental builds and build caching instead of the deep dive?

Reply:
Just checking on this - I see the comment is marked as resolved but the link didnt change - wanted to confirm you want to link to the deep dive and not the incremental training?